Gopkg.toml

```diff
@@ -30,7 +30,7 @@
 
 [[constraint]]
   name = "github.com/jessevdk/go-flags"
-  revision = "97448c91aac742cbca3d020b3e769013a420a06f"
+  revision = "4e64e4a4e2552194cf594243e23aa9baf3b4297e"
```

Comment:
Two things:
- As @mattChiaravalloti mentioned, I think we should update to the latest version possible, assuming it doesn't introduce any breakage
- Instead of a particular revision, let's use a tag-based constraint like the one we're using above for the go driver

Reply:
Cool, I changed the .toml file to take v1.4.0 (the most recent release) and updated with dep.